sdk/cognitiveservices/azure-cs-textanalytics/src/test/java/com.azure.cs.textanalytics/TextAnalyticsClientTestBase.java

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@mssfang Should these now be under `textanalytics` root folder and not `cognitiveservices` anymore?